edx_django_utils/monitoring/docs/how_tos/search_new_relic_nrql.rst

```diff
@@ -0,0 +1,6 @@
+Searching New Relic NRQL
+========================
+
+The search script `new_relic_nrql_search.py`_ is generally useful for searching NRQL (New Relic Query Language) in New Relic. It searches the NRQL in New Relic alert policies, and in New Relic dashboards. Use ``--help`` for more details.
```

Comment:
[inform] I removed the caveat around baseline alert conditions because that issue has been fixed by switching to a newer New Relic API in this PR: https://github.com/edx/edx-django-utils/pull/130